index.html

```diff
@@ -32,6 +32,18 @@ <h2>9:41</h2>
                 <img src="img/Wifi.png" alt="wifi">
                 <img src="img/Battery.png" alt="battery">
             </div>
+            <ul class="navbar">
```

Comment:
[ OPTIONAL ] The page has a horizontal scroll bar which means that some content is overflowing from the container. Kindly fix the scroll as it is not considered a good design.